_help/README.md

```diff
@@ -0,0 +1,96 @@
+# Maintaining Help Section
+
+The Help section of login.gov a slightly different from others. It is the core of the site mobile navigation. 
+
+Here are helpful hints for maintaining this section.
+
+## The _help files location
+For now, the help section is a direct subdirectory of the identity-site. They are located outside of the _pages directory. Our goal is to move this section to the _pages directory in a future pull request, where we can view the impact of such a move.
+
+## The Help.html template
+tbd
+
+## The Help.md landing page
+tbd
+
+## The nav__sidenav.html component
+tbd
+
+## Adding a new help page
+1. Navigate to en.yml.
+    1. Find the `help` variable. Add the new page under an existing category:
+        ```
+          get-started:
+            overview: Overview
+            new-page-slug: New page title
+        ```
+        > Here, the new page is added to the get-started collection of pages.
+    1. Add the new page to the `help_pages` variable:
+        ```
+          get-started:
+            overview: |-
+              You can add or delete email addresses from your login.gov Your Account page...
+            new-page-slug: |-
+              This is the content of the new Help page...
+        ```
+        Remember to combine the yaml literal operator and the strip operator `|-` to keep line breaks but remove ending empty lines from the content.
+
+        **DO NOT USE HTML.** Content should include markdown syntax only. This applies to headings, lists, images and links. If special styling is required for content _within_ the help article, consider using general CSS selectors. Do not use ids created by markdown, because these ids will be translated and will not apply to translated pages.
```

Comment:
>Do not use ids created by markdown, because these ids will be translated and will not apply to translated pages.

That's a handy "gotcha" that I hadn't considered before! 👍